velox/type/Conversions.h

```diff
@@ -57,6 +57,52 @@ struct Converter {
   }
 };
 
+/// This is based on Presto java's castToBoolean method.
+template <
+    class StringType,
+    typename = std::enable_if_t<
+        std::is_same_v<StringType, StringView> ||
+        std::is_same_v<StringType, folly::StringPiece> ||
+        std::is_same_v<StringType, std::string>>>
+static bool castToBoolean(const StringType& v) {
```

Comment:
It looks like the code can be simplified if this function is modified to take char* data + size_t size.